test/interpolated-string.tests.ts

```diff
@@ -148,6 +148,29 @@ world!";`);
                 Token.Punctuation.Semicolon]);
         });
 
+        it("verbatim with two interpolations (reverse)", async () => {
```

Comment:
You guys might want to consider moving to a snapshot model for grammar testing. I started doing this for our [Razor grammar](https://github.com/aspnet/AspNetCore-Tooling/tree/master/src/Razor/test/Microsoft.AspNetCore.Razor.VSCode.Grammar.Test/tests) and it's made testing super easy, non-manual and ensures we don't break anything.

- [Example](https://github.com/aspnet/AspNetCore-Tooling/blob/2fae94440496d57b69ddbc93e038b3509de1c931/src/Razor/test/Microsoft.AspNetCore.Razor.VSCode.Grammar.Test/tests/CodeBlock.ts#L47-L52) of a test
- [Example ](https://github.com/aspnet/AspNetCore-Tooling/blob/2fae94440496d57b69ddbc93e038b3509de1c931/src/Razor/test/Microsoft.AspNetCore.Razor.VSCode.Grammar.Test/tests/__snapshots__/GrammarTests.test.ts.snap#L3788-L3812) of the snapshot it generates / compares

We're using the [jest snapshot testing model](https://jestjs.io/docs/en/snapshot-testing) which takes care of updating/comparing the snapshots that we generate [with our custom factory](https://github.com/aspnet/AspNetCore-Tooling/blob/2fae94440496d57b69ddbc93e038b3509de1c931/src/Razor/test/Microsoft.AspNetCore.Razor.VSCode.Grammar.Test/tests/infrastructure/SnapshotFactory.ts).